StPetersburg/IVorlov7.xml

```diff
@@ -375,20 +370,18 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                            </supportDesc>
 
                            <layoutDesc>
-                              <layout>
-                                 <p>
+                              <layout>                                 
```

Comment:
nothing to say about the layout? columne/lines? which is the layout on these pages? if nothing is known then delete the element, the pages are already specified for the part

Reply:
I revised it a bit, it is difficult to be more precise because the catalogue description of Turaev is very vague and confusing.